tests/decorators/decorators.go

```diff
@@ -108,4 +108,8 @@ var (
 	NoFlakeCheck = Label("no-flake-check")
 	// FlakeCheck decorates tests that are dedicated to the check-tests-for-flakes test lane.
 	FlakeCheck = Label("flake-check")
+
+	// OncePerOrderedCleanup decorates Ordered tests to only cleanup after the last
+	// test in an Ordered container.
```

Comment:
Please clearly mention here that the use of this decorator is currently restricted to sig-network.

Reply:
If you agree, we could extend it to virtctl as well, see my last comment.